libs/helm/autocomplete/src/lib/hlm-autocomplete.ts

```diff
@@ -0,0 +1,202 @@
+import { BooleanInput } from '@angular/cdk/coercion';
+import {
+	booleanAttribute,
+	ChangeDetectionStrategy,
+	Component,
+	computed,
+	ElementRef,
+	forwardRef,
+	inject,
+	input,
+	linkedSignal,
+	output,
+	signal,
+} from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { NgIcon, provideIcons } from '@ng-icons/core';
+import { lucideChevronDown, lucideSearch } from '@ng-icons/lucide';
+import { BrnCommandEmpty } from '@spartan-ng/brain/command';
+import { BrnDialogState } from '@spartan-ng/brain/dialog';
+import { ChangeFn, TouchFn } from '@spartan-ng/brain/forms';
+import { BrnPopover, BrnPopoverContent } from '@spartan-ng/brain/popover';
+import {
+	HlmCommand,
+	HlmCommandEmpty,
+	HlmCommandGroup,
+	HlmCommandItem,
+	HlmCommandList,
+	HlmCommandSearch,
+	HlmCommandSearchInput,
+} from '@spartan-ng/helm/command';
+import { HlmIcon } from '@spartan-ng/helm/icon';
+import { HlmPopoverContent } from '@spartan-ng/helm/popover';
+import { HlmAutocompleteTrigger } from './hlm-autocomplete-trigger';
+
+export const HLM_AUTOCOMPLETE_VALUE_ACCESSOR = {
+	provide: NG_VALUE_ACCESSOR,
+	useExisting: forwardRef(() => HlmAutocomplete),
+	multi: true,
+};
+
+let nextId = 0;
+
+@Component({
+	selector: 'hlm-autocomplete',
+	imports: [
+		BrnPopover,
+		BrnPopoverContent,
+		HlmPopoverContent,
+
+		HlmCommand,
+		HlmCommandSearch,
+		HlmCommandSearchInput,
+		HlmCommandList,
+		HlmCommandGroup,
+		HlmCommandItem,
+		BrnCommandEmpty,
+		HlmCommandEmpty,
+
+		HlmAutocompleteTrigger,
+
+		NgIcon,
+		HlmIcon,
+	],
+	providers: [HLM_AUTOCOMPLETE_VALUE_ACCESSOR, provideIcons({ lucideSearch, lucideChevronDown })],
+	template: `
+		<brn-popover
+			[align]="'start'"
+			[autoFocus]="'first-heading'"
+			[closeOnOutsidePointerEvents]="true"
+			[state]="_popoverState()"
+			(stateChanged)="_stateChanged($event)"
+			sideOffset="5"
+			closeDelay="100"
+		>
+			<hlm-command class="border-border w-full border">
+				<hlm-command-search class="h-full" hlmAutocompleteTrigger [disabled]="!_filter()">
```

Comment:
The input height is larger than all other controls like input, select, combobox etc.. and there is no box shadow:

<img width="351" height="61" alt="image" src="https://github.com/user-attachments/assets/bb49946e-32a2-4dd9-93a1-19b968497153" />

Is this intentional? Or should they align?

Reply:
<img width="544" height="148" alt="CleanShot 2025-08-27 at 13 14 45" src="https://github.com/user-attachments/assets/8850d5b3-f245-41aa-9fe2-fb8a3d930694" />